### PR TITLE
Update React no-danger error message to contain `SafeMarkdown` 

### PR DESCRIPTION
**Pull request by nkiruka**
# Description
Update React no-danger error message to contain `SafeMarkdown`

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by Hamms** [comment]
Updated formatter to be more of a preprocessor to adjust the results data before passing it off to the default formatter.

Old output vs new output looks like this:

```diff
@@ -1,8 +1,8 @@
 yarn run v1.19.2
-$ eslint --ext .js,.jsx src test
+$ eslint --format ./.eslintCustomMessagesFormatter.js --ext .js,.jsx src test
 
 /home/elijah/projects/code.org/code-dot-org/apps/src/applab/ImportScreensDialog.jsx
-  153:13  error  Dangerous property 'dangerouslySetInnerHTML' found  react/no-danger
+  153:13  error  Dangerous property 'dangerouslySetInnerHTML' found. Please use the SafeMarkdown React Component instead if possible  react/no-danger
 
 ✖ 1 problem (1 error, 0 warnings)
```

File: tools/hooks/lint.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def run_rubocop(files)
 end
 
 def run_eslint_apps(files)
-  run("./node_modules/.bin/eslint -c .eslintrc.js #{files.join(' ')}", APPS_DIR)
+  run("./node_modules/.bin/eslint -c .eslintrc.js -f ./.eslintCustomMessagesFormatter.js #{files.join(' ')}", APPS_DIR)
```
Comment (Hamms) [inline tools/hooks/lint.rb:59]: I don't love that this needs to be updated in two places, but I think fixing that is outside the scope of this PR

File: apps/.eslintCustomMessagesFormatter.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * @type {Object.<string, function>}
+ */
+const RULE_MESSAGE_OVERRIDES = {
+  // for 'dangerouslySetInnerHTML' warnings, suggest our custom SafeMarkdown
+  // component as an alternative and include the original error message for
+  // clarity.
+  'react/no-danger': messageObject =>
+    `${
+      messageObject.message
+    }. Please use the SafeMarkdown React Component instead if possible.`
```
Comment (daynew) [inline apps/.eslintCustomMessagesFormatter.js:29]: Would it be helpful if the message provided a link to where they can learn how to use the SafeMarkdown React Component?

**Review by daynew — APPROVED** [review]
LGTM. Just one minor documentation comment.

File: apps/.eslintCustomMessagesFormatter.js (hunk 2 above)
Comment (Hamms) [reply apps/.eslintCustomMessagesFormatter.js:29]: ooh, I like that! Added https://codedotorg.atlassian.net/browse/FND-967 to track this
